gtars/tests/test.rs

```diff
@@ -21,6 +21,14 @@ fn path_to_sorted_small_bed_file() -> &'static str {
     "tests/data/test_sorted_small.bed"
 }
 
+#[fixture]
+fn path_to_small_bam_file() -> &'static str {
+    "tests/data/test1_sort_dedup.bam"
+}
+
+#[fixture]
+fn path_to_chrom_sizes_file() -> &'static str {"tests/hg38.chrom.sizes"}
```

Comment:
Can you run `cargo fmt` ? Makes things nice and consistent

Reply:
done